samples/SampleSender/Program.cs

```diff
@@ -15,7 +15,12 @@ public class Program
 
         public static void Main(string[] args)
         {
-            SendMessagesToEventHub(100).Wait();
+            MainAsync(args).GetAwaiter().GetResult();
+        }
+
+        private static async Task MainAsync(string[] args)
+        {
+            await SendMessagesToEventHub(100).ConfigureAwait(false);
```

Comment:
@SeanFeldman - While it may be considered a best practice, we believe it can be confusing for the Getting Started articles. Users may not know that this is not related to Event Hubs specifically, but instead the .NET async server paradigm.